src/appleseed/renderer/api/rendering.h

```diff
@@ -45,7 +45,6 @@
 #include "renderer/kernel/rendering/masterrenderer.h"
 #include "renderer/kernel/rendering/nulltilecallback.h"
 #include "renderer/kernel/rendering/progressive/progressiveframerenderer.h"
-#include "renderer/kernel/rendering/scenepicker.h"
```

Comment:
scenepicker.h brings ShadingPoint and that brings OSL back again.
We cannot remove the OSL includes from shadingpoint. The class holds an instance of OSL::ShaderGlobals.

ScenePicker is not being used in any of our plugins as far as I know. It's only used in studio.